x-pack/plugin/core/src/test/java/org/elasticsearch/license/ClusterStateLicenseServiceClusterTests.java

```diff
@@ -33,8 +33,6 @@ private Settings.Builder nodeSettingsBuilder(int nodeOrdinal, Settings otherSett
     }
 
     public void testClusterRestartWithLicense() throws Exception {
-        wipeAllLicenses();
```

Comment:
Wiping the licenses before the test starts doesn't make sense if we start with an empty cluster (i.e. no nodes).